### PR TITLE
Version 2.6.0

### PR DESCRIPTION
**Pull request by mpeyfuss**
Updated the following
- `Makefile` - installs libraries without commits so can be batched into one commit
- `foundry.toml` - remove unneeded items and to switch to the default EVM version
- `Doppelganger.sol` and `CollectorsChoice.sol` - implements ERC-4906

### CODE REVIEW

File: foundry.toml
Comment (bstrauss23) [inline foundry.toml:1]: Not sure why all of these params aren't needed, but it looks like you are removing a lot of them. I'll just have to trust ya!

File: src/doppelganger/CollectorsChoice.sol [hunk 1]
```diff
@@ -169,7 +169,7 @@ contract CollectorsChoice is ERC1967Proxy {
 
         store.tokens[tokenId] = tokenUriIndex;
 
-        emit URIChanged(msg.sender, tokenId, store.uris[tokenUriIndex]);
+        emit MetadataUpdate(tokenId);
```
Comment (bstrauss23) [inline src/doppelganger/CollectorsChoice.sol:172]: is URIChanged more accurate language here?

File: src/doppelganger/Doppelganger.sol [hunk 2]
```diff
@@ -143,7 +143,7 @@ contract Doppelganger is ERC1967Proxy {
 
         store.tokens[tokenId] = tokenUriIndex;
 
-        emit URIChanged(msg.sender, tokenId, store.uris[tokenUriIndex]);
+        emit MetadataUpdate(tokenId);
```
Comment (bstrauss23) [inline src/doppelganger/Doppelganger.sol:146]: Same comment here about language

**Review by bstrauss23 — COMMENTED** [review]
See comments first.

File: foundry.toml
Comment (mpeyfuss) [reply foundry.toml:1]: Basically everything should be fine by default.

File: src/doppelganger/CollectorsChoice.sol (hunk 1 above)
Comment (mpeyfuss) [reply src/doppelganger/CollectorsChoice.sol:172]: MetadataUpdate will trigger automatic refreshed on OpenSea and is specified by ERC-4906 so this is much better to use.

**Review by bstrauss23 — APPROVED** [review]

File: foundry.toml
Comment (mpeyfuss) [reply foundry.toml:1]: added a few more back in

**Pull request merged**